pkg/charts/charts.go

```diff
@@ -24,6 +32,106 @@ type Chart struct {
 	Files     []*chartutil.BufferedFile
 }
 
+// LoadChart returns a new Chart object that contains all the files of the
+// specified chart. The chart's files are loaded from the virtual filesystem
+// using the Helm's chartutil.LoadFiles() helper function.
+func LoadChart(chartName string) (*helmChart.Chart, error) {
+	var vfiles []*chartutil.BufferedFile
+
+	// retrieve all the files of a chart located at linkerdChartRootRid
+	walkVFS := func() error {
+		files, err := readVirtualFiles(linkerdChartRootDir, chartName)
+		if err != nil {
+			return err
+		}
+
+		for path, file := range files {
+			data, err := ioutil.ReadAll(file)
+			if err != nil {
+				return err
+			}
+
+			filename := path
+			if strings.HasPrefix(path, chartName) {
+				filename = path[len(chartName)+1:]
+			}
+
+			vfiles = append(vfiles, &chartutil.BufferedFile{
+				Name: filename,
+				Data: data,
+			})
+		}
+
+		return nil
+	}
+
+	if err := walkVFS(); err != nil {
+		return nil, err
+	}
+
+	return chartutil.LoadFiles(vfiles)
+}
+
+// LoadDependencies loads all the dependent subcharts of the specified chart.
+// It relies on LoadChart to load the files and metadata of the chart from the
+// VFS.
+func LoadDependencies(chartName string) ([]*chart.Chart, error) {
+	chart, err := LoadChart(chartName)
+	if err != nil {
+		return nil, err
+	}
+
+	return chart.Dependencies, nil
+}
```

Comment:
In the patch I sent to @Pothulapati, I didn't want to call `LoadChart()` in `install.go` directly, to avoid confusion with the existing `l5dchart.Chart` struct.